_includes/header.html

```diff
@@ -1,217 +1,181 @@
-<div class="syn-styles-supported syn-header-wrapper">
-  <header class="syn-page-header">
-    <div class="syn-page-header-nav-backdrop" onclick="['nav-open'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))"></div>
-    <div class="syn-page-header-content syn-flex justify-space-between">
-      <div class="syn-flex align-center">
-        <button
-          class="syn-btn icon text grow syn-hidden-lg"
-          onclick="['nav-open', 'nav-collapsed'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))"
-        >
-          <svg width="16px" height="16px" viewBox="0 0 12 12">
-            <path
-              fill="currentColor"
-              d="M0,2 L12,2 L12,3.344 L0,3.344 L0,2 Z M0,6.67 L0,5.33 L12,5.33 L12,6.67 L0,6.67 Z M0,10 L0,8.656 L12,8.656 L12,10 L0,10 Z"
-            />
-          </svg>
-        </button>
-        <a class="syn-flex" href="/">
-          {% if site.isLocalDev or site.enterprise %}
-          <img
-            class="syn-ml-8"
-            src="{{ site.baseurl }}/images/algo-purple.svg"
-            alt="Algorithmia Logo"
-            style="max-height: 24px; max-width: 100%"
-          />
-          {% else %}
-          <img
-            class="syn-ml-8"
-            src="/v1/config/logo"
-            alt="Algorithmia Logo"
-            style="max-height: 24px; max-width: 100%"
-          />
-          {% endif %}
+<header class="syn-styles-supported supports-synapse-typography syn-page-header">
+  <div class="utility-header syn-visible-lg">
+    <div class="syn-pr-32 syn-pv-8 syn-flex flex-row justify-end">
+      <div class="hide-if-signed-in">
+        <a href="https://teams.algorithmia.com/signin" data-e2e="header-signin-link" class="syn-link syn-font-weight-bold syn-text-secondary syn-text-sm">
+          Sign in
         </a>
-        <div
-          class="syn-page-header-divider vertical syn-ml-16 syn-mr-16 syn-hidden-lg syn-hidden-md syn-hidden-sm"
-          id="before-brand-title"
-        ></div>
-        <span class="syn-font-mono syn-caption syn-text-tertiary syn-mb-0 syn-hidden-lg syn-hidden-md syn-hidden-sm" id="brand-title"></span>
-      </div>
+      </div> 
       <div>
-        {% unless site.enterprise %}
-        <a href="/product" class="syn-link-no-decoration show-auth show-unauth">
-          <button class="syn-btn text syn-visible-lg">Product</button>
-        </a>
-        <a href="/pricing" class="syn-link-no-decoration show-auth show-unauth">
-          <button class="syn-btn text syn-visible-lg syn-ml-16">
-            Pricing
-          </button>
+        <a href="https://teams.algorithmia.com/contact" class="syn-link syn-font-weight-bold syn-text-secondary syn-text-sm syn-ml-24">
```

Comment:
We don't have a contact page on teams. Will need to remain on the root domain for now.
```suggestion
        <a href="/contact" class="syn-link syn-font-weight-bold syn-text-secondary syn-text-sm syn-ml-24">
```